chapter_deep-learning-computation/parameters.md

```diff
@@ -52,7 +52,7 @@ import torch
 from torch import nn
 
 net = nn.Sequential(nn.Linear(4, 8), nn.ReLU(), nn.Linear(8, 1))
-X = torch.randn(2, 4)
```

Comment:
Can we do an x-ray of the entire pytorch implementations to see which `torch.randn` has to be changed?

Reply:
Sure! I'll do that and send a PR if any changes are needed.